posts.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
Delete this file doesnt look like you need it since you're using the one in the theme dir

Reply:
done.